v2/features/connection/hooks.ts

```diff
@@ -82,19 +83,25 @@ export function useInactiveListener(suppress = false) {
   }, [active, error, suppress, activate]);
 }
 
-export const useENS = (address: string, provider: any, chainId: number | undefined) => {
-  // const { provider } = Connection.useContainer();
+export const useENS = (
+  address: string | null | undefined,
```

Comment:
@amatureApe in general, let's try to avoid `any` unless there is no other way. With the right type as input we're forced to handle every possible scenario below, makes for a safer implementation.

Reply:
Ok makes sense, thanks.